css/style.css

```diff
@@ -0,0 +1,3 @@
+header {
+  color: lightblue;
+}
```

Comment:
- [ ] It is the best practice to use class selector instead of element selector to style your `h1` element. Kindly use the class name from your `h1` element to style its content.